packages/go_router/example/lib/books/src/screens/books.dart

```diff
@@ -105,10 +102,8 @@ class _BooksScreenState extends State<BooksScreen>
     switch (index) {
       case 1:
         context.go('/books/new');
-        break;
       case 2:
         context.go('/books/all');
-        break;
       case 0:
       default:
```

Comment:
the default should probably be removed

Reply:
That doesn't seem related to applying the new lints and I'd like to leave that out of this already sizable PR.